### PR TITLE
chore: updated documentation

### PR DESCRIPTION
**Pull request by shaimendel**
- [ ] Ready for review
- [ ] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Updated README.md.
Look at https://github.com/snyk/nodejs-runtime-agent/tree/chore/update_readme to view the README.md visually

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,15 @@ The `config` object supports the following options:
 |--------------------|-----------|------------------------------------------|-------------------------------------------------------------------------|
 | `projectId`        | `String`  |                                          | The Snyk project ID matching to your application.                       |
 | `enable`           | `Boolean` | `true`                                   | Set to `false` to disable the agent.                                    |
-| `url`              | `String`  | `https://homebase.snyk.io/api/v1/beacon` | Override to have the agent report its beacons to an alternative server. |
+
+Advanced `config` options:
```
[the remark this answers is not in the document]
Comment (shaimendel) [reply README.md]: Currently not, it's either removing it or leaving it

[the remark this answers is not in the document]
Comment (Shesekino) [reply README.md]: then in my opinion remove it until we can extract it

File: README.md [hunk 2]
```diff
@@ -11,10 +16,17 @@ The `config` object supports the following options:
...
-| `beaconIntervalMs` | `Number`  | `60000`                                  | Beacon interval duration in milliseconds.                               |
+
+Advanced `config` options:
+
+| Key                  | Type      | Default value                                               | Purpose                                                                                    |
+|----------------------|-----------|-------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| `beaconUrl`          | `String`  | `https://homebase.snyk.io/api/v1/beacon`                    | Override to have the agent report to an alternative server.                                |
+| `snapshotUrl`        | `String`  | `https://homebase.snyk.io/api/v2/snapshot/<projectId>/node` | Override to have the agent pull vulnerable function snapshots from an alternative server.  |
+| `beaconIntervalMs`   | `Number`  | `60000`                                                     | Report interval frequency in milliseconds.                                                 |
```
Comment (Shesekino) [inline README.md]: not interval frequency, just frequency

File: README.md [hunk 3]
```diff
@@ -11,10 +16,17 @@ The `config` object supports the following options:
...
+| `snapshotIntervalMs` | `Number`  | `3600000`                                                   | Snapshot retrieval interval frequency in milliseconds.                                     |
```
Comment (Shesekino) [inline README.md]: same

**Pull request merged**